docs/guides/postinstall.md

```diff
@@ -91,6 +91,8 @@ The steps to be followed vary depending upon the initramfs module loading mechan
 
 The Touch Bar can be set up by running [this script](../tools/touchbar.sh) **in Linux** using `bash /path/to/script`. Make sure your Linux kernel and macOS is updated before running this script.
 
+As a secondary option, if you want to try something new and you're using Arch Linux or EndeavourOS, you could install `touchbard` package and use that instead. For more information, visit [touchbard repository](https://github.com/NoaHimesaka1873/touchbard).
```

Comment:
Isn't it available for Manjaro?

Reply:
I didn't upload the package to Manjaro repository. I shall upload and add Manjaro too.